### PR TITLE
Table component

### PR DESCRIPTION
**Pull request by hookah96**

### CODE REVIEW

File: src/components/Table/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      hooks.allColumns.push((columns) => [
+        // Let's make a column for selection
+        ...columns
+      ])
+      hooks.useInstanceBeforeDimensions.push(({ headerGroups }) => {
+        // fix the parent group of the selection button to not be resizable
+        const selectionGroupHeader = headerGroups[0].headers[0]
+        selectionGroupHeader.canResize = false
+      })
+    }
```
Comment (nikoskleidis) [inline src/components/Table/index.js:34]: Is this code needed?

Comment (hookah96) [reply src/components/Table/index.js:34]: I just checked it. No it isn't.

File: src/components/Table/PatientTable.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  const age = dayjs().from(dayjs(birthDate), true)
+  return (
+    <div>
+      {firstName} {lastName}
+      <div className={subrow}>{age} old</div>
+    </div>
+  )
+}
+
+console.log()
```
Comment (nikoskleidis) [inline src/components/Table/PatientTable.js:26]: Please remove this line

File: src/components/Table/PatientTable.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+      {lastVisit}
+      <div className={subrow}>{user}</div>
+    </div>
+  )
+}
+
+function ActionsColumn() {
+  return (
+    <div>
+      <NavLink to="patients/details/:id" className={viewAction}>
```
Comment (nikoskleidis) [inline src/components/Table/PatientTable.js:62]: Id needs to be replaced with the actual id of the patient that we are rendering

**Pull request merged**